cirq/api/google/v2/metrics.proto

```diff
@@ -0,0 +1,79 @@
+syntax = "proto3";
+
+package cirq.api.google.v1;
+
+option java_package = "com.google.cirq.api.google.v2";
+option java_outer_classname = "CalibrationProto";
+option java_multiple_files = true;
+
+// Performance metrics for a quantum processor.
+message Metrics {
```

Comment:
It's confusing that a `Metrics` contains many `MetricCollection`s instead of many `Metric`s.

Maybe `NamedMetricCollections`?

Reply:
Ugh.  That's like java ;)  
 How about PerformanceSnapshot?

Reply:
Yeah that's better.